sdk/script/management-sdk.ts

```diff
@@ -257,8 +251,15 @@ class AccountManager {
         });
     }
 
-    public promote(appName: string, sourceDeploymentName: string, destDeploymentName: string): Promise<void> {
-        return this.post(urlEncode `/apps/${appName}/deployments/${sourceDeploymentName}/promote/${destDeploymentName}`, /*requestBody=*/ null, /*expectResponseBody=*/ false)
+    public patchRelease(appName: string, deploymentName: string, packageInfo: PackageInfo): Promise<void> {
+        var requestBody: string = JSON.stringify({ packageInfo: packageInfo });
+        return this.patch(urlEncode `/apps/${appName}/deployments/${deploymentName}/release`, requestBody, /*expectResponseBody=*/ false)
+            .then(() => null);
+    }
+
+    public promote(appName: string, sourceDeploymentName: string, destDeploymentName: string, packageInfo: PackageInfo): Promise<void> {
```

Comment:
Can we rename the `packageInfo` parameter to `updatedMetadata`? Also, I know this wasn't part of your PR, but could we rename `destDeploymentName` to `destinationDeploymentName`? We aren't using abbreviations anywhere else, so it would be nice to be consistent.